tests/Oryx.BuildImage.Tests/BaseImage/BaseImageTest.cs

```diff
@@ -82,7 +82,6 @@ public BaseImageTest(ITestOutputHelper output) : base(output)
             var appOutputDir = "/tmp/output";
             var manifestFile = $"{appOutputDir}/{FilePaths.BuildManifestFileName}";
             var script = new ShellScriptBuilder()
-                .AddDefaultTestEnvironmentVariables()
```

Comment:
+1. Is the intention to use the prod storage account here or the newly created private storage account?
Some tests may fail during validation/nightly pipelines if we take the route of using the prod storage account for these tests since version specific tarballs may not be released to prod yet.

Reply:
I have explained the reason here:
https://github.com/microsoft/Oryx/pull/1878#discussion_r1191447904

Please let me know your thoughts. thanks

Reply:
I like it. Thanks for making the change